pages/brands.yaml

```diff
@@ -56,36 +56,36 @@ areas:
               type: Link
               params:
                 pageId: new-brand # The page id of the new brand page.
-      # A List block to render the following brands.
```

Comment:
We no longer need to set lists to state if we use a table so we should remove that

Reply:
Change made in 48ac3ac2c6759a62f2cdebd3ff80e322df83603e